crml/staking/src/lib.rs

```diff
@@ -1473,6 +1467,24 @@ impl<T: Trait> Module<T> {
 		maybe_new_validators
 	}
 
+	/// Compute current `total_payout` and `max_payout` for specific era duration
+	pub fn current_total_payout(total_issuance: RewardBalanceOf<T>) -> (RewardBalanceOf<T>, RewardBalanceOf<T>) {
```

Comment:
If it's a security concern to expose this, we can make it private and make another method that wraps it and guard it behind #[test]

Reply:
it's ok since it's read only